packages/specification/schemas/components/_extensions/EPSTaskBusinessStatus.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
```

Comment:
The description for this should be: 
"The intended status of the prescription as a result of the dispense notification."